src/components/Form/story.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Form from '.';
+import Readme from './README.md';
+
+storiesOf('Uncategorized/Form', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add(
+    'Overview',
+    () => (
+      <Form
+        initialValues={{ nameInput: 'james r. cat' }}
+        // eslint-disable-next-line no-alert
+        onSubmit={(formValues) => alert(JSON.stringify(formValues))}
+        validationSchema={{
+          nameInput: {
+            message: 'No empty strings',
+            validate: (value) => value !== '',
+          },
+        }}
+      >
+        {({
+          formState,
+          formValues,
+          handleChange,
+          handleBlur,
+          handleCustom,
```

Comment:
The name `handleCustom` is confusing to me. Is it a change handler, blur handler, submit handler?
Maybe something along the lines of `processFieldValue`?

Would it be better to have it take and update the entire form state? I'm thinking of cases where fields would depend on each other, like if a user chose "February" and we had to reset or modify a date from 31 to 28.